_posts/2023-16-15-Second-Issue-Newsletter

```diff
@@ -0,0 +1,292 @@
+---
+layout: post
+categories: newsletter
+title: "The OBO Foundry Newsletter: Advancing Bio-Ontologies for Enhanced Data Integration"
+date: 2023-09-15
+author: 
+  - Leila Kiani
+---
+
+
+Welcome to the second issue of the OBO Foundry Newsletter! We are delighted to continue our journey of advancing bio-ontologies and promoting enhanced data integration within our community. From general announcements to training opportunities and event updates, we strive to keep you up-to-date with the latest advancements and activities within the OBO Foundry.
+
+One of our primary goals is to foster collaboration and celebrate the contributions of our community members. In this edition, we will shine a spotlight on some of our current members, highlighting their achievements and fostering a sense of connection and cooperation.
+
+We encourage you to share your thoughts and ideas with the OBO Operations Committee, as your feedback helps shape the future of the OBO Foundry.
+
+Thank you for being part of our community. We hope you enjoy this second edition and find it enriching and informative.
+
+Best regards,  [The OBO Foundry Operations Committee.](https://obofoundry.org/docs/OperationsCommittee.html)
+
+
+---
+
+## In Memoriam: Remembering Michael Ashburner
+
+
+
+<img src="https://obofoundry.org/images/ashburner.png" style="max-height: 250px" alt="Michael Ashburner" />
+
+
+
+
+We bid farewell to Michael Ashburner, a distinguished biologist and an unwavering advocate for open science. Ashburner, a pioneering figure in genetics, bioinformatics, and the OBO Foundry, passed away on July 10, 2023, at 81. His visionary leadership at EMBL-EBI left an indelible mark on bioinformatics. He sequenced the Drosophila genome, co-founding FlyBase and shaping ontologies like GO and ChEBI. Advocating for open science, he championed genome accessibility and open publishing. Honored with awards such as the Gregor Mendel Medal, his legacy inspires future scientists. Our heartfelt condolences go out to his family, friends, and colleagues.
+
+ [The OBO Foundry Operations Committee.](https://obofoundry.org/docs/OperationsCommittee.html)
+
+
+---
+
+## Highlights 
+
+
+### Passing the OBO Dashboard will become mandatory on January 1st 2024
```

Comment:
add superscripts to nicely stylize dates
 
```suggestion
### Passing the OBO Dashboard will become mandatory on January 1<sup>st</sup> 2024
```

Reply:
Github does not want to accept your commit, sorry:

![Uploading image.png…]()

Reply:
I renamed it as this is the first comment I saw.